src/main/pages/che-6/workspace-admin/projects.adoc

```diff
@@ -35,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
  ]
 ----
 
-Once a project is saved into a workspace config, the IDE will check if the project exists on a file system, and if not, import it using `source.location` URL. It happens at IDE initialization stage.
+Once a project is saved into a workspace configuration, the IDE will check if the project exists on a file system.  If the project does not exist on the file sytem, you can import it using the `source.location` URL. This happens during the IDE initialization stage.
```

Comment:
You use "a filesystem" here followed by "the filesystem" -- which applies best? I'd use whichever is most accurate in both cases.

Reply:
Since the original text has "a", I think that it should be "a"